registration-processor/registration-processor-info-storage-service/src/main/java/io/mosip/registration/processor/packet/storage/utils/PacketManagerService.java

```diff
@@ -255,7 +255,7 @@ public void addOrUpdateTags(String id, Map<String, String> tags) throws ApisReso
     }
 
 	@SuppressWarnings("unchecked")
-	public boolean deleteTags(String id, List<String> tags)
+	public String deleteTags(String id, List<String> tags)

```

Comment:
Lets this method also be void, if there is any error anyway the exception will be thrown rt ?

Reply:
Fix: changed return type to void